pkg/cluster/internal/providers/docker/provision.go

```diff
@@ -245,6 +245,8 @@ func runArgsForNode(node *config.Node, clusterIPFamily config.ClusterIPFamily, n
 		"--volume", "/lib/modules:/lib/modules:ro",
 		// propagate KIND_EXPERIMENTAL_CONTAINERD_SNAPSHOTTER to the entrypoint script
 		"-e", "KIND_EXPERIMENTAL_CONTAINERD_SNAPSHOTTER",
+		// unset container=docker and fake the value, for debugging https://github.com/kubernetes-sigs/kind/issues/2409
+		"-e", "container=podman",
```

Comment:
This seems to work, but not sure why.

cc @ktock 
https://github.com/kubernetes-sigs/kind/pull/2354 might be related?